config.go

```diff
@@ -338,3 +338,31 @@ func nodejs(c *Config) error {
 
 	return nil
 }
+
+// python config.
+func python(c *Config) {
+	if c.Proxy.Command == "" {
+		c.Proxy.Command = "python app.py"
+	}
+
+	// Only add build & clean hooks if a requiremnts.txt exists
+	if !util.Exists("requirements.txt") {
+		return
+	}
+
+	// Set PYTHONPATH env
+	if c.Environment == nil {
+		c.Environment = config.Environment{}
+	}
+	c.Environment["PYTHONPATH"] = ".pypath/"
+
+	// Copy libraries into .pypath/
+	if c.Hooks.Build == "" {
+		c.Hooks.Build = `mkdir -p .pypath/ && pip install -r requirements.txt -t .pypath/`
```

Comment:
@arikfr That's def

Working on this PR raised two questions:
* How "involved" should `up` be in the build process ?
    * Do you expect most users to write custom `hooks.build` or should "vanilla" `up` work 99% of the time ? (to preserve the "magic" of `up`)
    * Or should there be something similar to heroku buildpacks ? (detect project/language type with build logic/scripts to produce a "slug" (`.tar.gz`))
    * Should we go as far as doing `docker` builds for users ? (to cross compile / isolate deps and increase reproducibility)
* Would a `up local` command make sense ?